src/core/network-openssl.c

```diff
@@ -374,6 +378,20 @@ static gboolean irssi_ssl_init(void)
 	SSL_load_error_strings();
 	OpenSSL_add_all_algorithms();
 #endif
+	store = X509_STORE_new();
+	if (store == NULL) {
+		g_error("Could not initialize OpenSSL: X509_STORE_new() failed");
+		return FALSE;
+	}
+
+	success = X509_STORE_set_default_paths(store);
+	if (success == 0) {
+		g_error("Could not load default certificates");
```

Comment:
This was copied from a line that had `g_warning()` originally.

`g_error()` is fatal and results in `abort()`. Probably not the wisest thing to do if there's no system-wide cert store. 

I'd also remove the `return FALSE` below to let it set ssl_inited.

(Hard-failing on `X509_STORE_new` errors seems fine, it's mostly malloc failures)